docs/baking.md

```diff
@@ -37,16 +37,13 @@ The most convenient way to orchestrate all these binaries is to use the `tezos-b
 package, which provides predefined services for running baking instances on different
 networks.
 
-To install them, run the following commands:
+First, add package repository:
```

Comment:
With this change it almost looks like the `Update and install packages:` is only for the `On Raspberry Pi OS` due to the headings:
![Screenshot_20210507_151307](https://user-images.githubusercontent.com/12936619/117454864-e0dbba00-af46-11eb-8fd0-cac81d68da4f.png)

Now that there's a "section" that contains just the repository addition, I suggest going with:

    #### First, add the package repository:
    
    <a name="ubuntu"></a>
    On Ubuntu:
    ```
    # Add PPA with Tezos binaries
    sudo add-apt-repository ppa:serokell/tezos
    ```
    
    <a name="pios"></a>
    On Raspberry Pi OS:
    ```
    # Install software properties commons
    sudo apt-get install software-properties-common
    # Add PPA with Tezos binaries
    sudo add-apt-repository 'deb http://ppa.launchpad.net/serokell/tezos/ubuntu bionic main'
    sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 37B8819B7D0D183812DCA9A8CE5A4D8933AE7CBB
    ```
    
    #### Update and install packages:

instead.